experiments/enableTTL/README.md

```diff
@@ -0,0 +1,188 @@
+# Effect of `enableTTL`
+
+Dispatcher versions 4.1.11 and greater can be configured to respect a time-to-live (TTL) based content "timeout". TTL headers (`Cache-Control` or `Expires`) must be included in responses sent from the publish tier in order for the dispatcher to know how long a cached file should "live" for in the cache. When this configuration is enabled (and reasonable TTL values are set), there is no longer a need for a dispatcher flush agent.
+
+## Compatibility
+
+The `enableTTL` configuration can be used with both AEM 6.5 and AEM as a Cloud Service. However, the replication framework from previous versions of AEM is no longer used to publish pages on AEM as a Cloud Service (read the details here: [Content Distribution](https://docs.adobe.com/content/help/en/experience-manager-cloud-service/core-concepts/architecture.html#content-distribution)). You may also find the AEMaaCS [Caching documentation](https://docs.adobe.com/content/help/en/experience-manager-cloud-service/implementing/content-delivery/caching.html) of interest, which goes into more detail about the differences.
+
+## Setup
+
+Ensure you have worked through the [Getting set up](../../README.md#getting-set-up) steps first.
+
+For this experiment, you will need to install the [ACS AEM Commons](https://adobe-consulting-services.github.io/acs-aem-commons/) package on your local **Publish** instance. We'll be using the [Dispatcher TTL](https://adobe-consulting-services.github.io/acs-aem-commons/features/dispatcher-ttl/index.html) feature to set the `max-age` of our content via an OSGi configuration. You can download the latest release here (v4.8.4 at the time of writing): [adobe-consulting-services.github.io/acs-aem-commons/](https://adobe-consulting-services.github.io/acs-aem-commons/)
+
+Next, install it on your local publish instance via Package Manager (you may need to [sign in first](http://localhost:4503/libs/granite/core/content/login.html)): [localhost:4503/crx/packmgr/index.jsp](http://localhost:4503/crx/packmgr/index.jsp)
+
+To verify that the package is installed correctly and is aware of our `DispatcherMaxAgeHeaderFilter` [OSGi config](https://github.com/adobe/aem-dispatcher-experiments/blob/enableTTL/aem-project/ui.apps/src/main/content/jcr_root/apps/dispatchertester/config/com.adobe.acs.commons.http.headers.impl.DispatcherMaxAgeHeaderFilter-paths-set-1.xml#L5) (included in this repo's `aem-project/`, which sets the `max.age` of `/content/dispatchertester/*` content to 60 seconds), issue the following request with cURL:
+
+```
+curl -H "Server-Agent: Communique-Dispatcher" -D - http://localhost:4503/content/dispatchertester/us/en.html -o /dev/null
+```
+
+Ensure the `Cache-Control: max-age=60` header is included in the response before continuing. 
```

Comment:
It is assumed, if they have gotten this far, that checking for headers should be elementary?  Makes sense, but thought I'd ask.

Reply:
The cURL command I include above makes them pretty straightforward to spot. Here's example output:

```
Date: Fri, 02 Oct 2020 01:38:59 GMT
Cache-Control: max-age=60
X-Content-Type-Options: nosniff
Content-Type: text/html;charset=utf-8
Transfer-Encoding: chunked

100  6277    0  6277    0     0  10973      0 --:--:-- --:--:-- --:--:-- 10973
```